manager/controllers/app/Makefile

```diff
@@ -3,7 +3,7 @@ include $(ROOT_DIR)/Makefile.env
 
 .PHONY: test
 test: *.go
-	go test -v
+	go test $(TEST_OPTIONS)
```

Comment:
should we add `$(TEST_OPTIOS)` to the line 15 ?

Reply:
added